content/sensu-go/6.6/operations/deploy-sensu/generate-certificates.md

```diff
@@ -196,8 +196,7 @@ For example, the directory listing of /etc/sensu/tls on backend-1 should include
 /etc/sensu/tls/
 ├── backend-1-key.pem
 ├── backend-1.pem
-├── ca.pem
-└── ca-key.pem
+└── ca.pem
```

Comment:
Shouldn't /etc/sensu/tls/ still contain ca-key.pem at this point? The order we are describing is:
- generate the CA certs/keys
- copy the folder for each backend
- generate backend-specific versions of the CA certs/keys with the commands above
- make sure only the `sensu` user can access the CA certs/keys

By my reading, it seems the ca-key.pem file needs to be in /etc/sensu/tls/ until that process is completed.